Dockerfile

```diff
@@ -1,21 +1,35 @@
 # sections ordered from least likely to change to most
 FROM cfmeqe/sel_base:latest
+
+ENV CHROME_DRIVER_VERSION 2.21
```

Comment:
>I presume this sets a default? 

Yep, it allows us to do custom containers and make version bumps clear

>Both the chrome driver and selenium bump seemed to work with the framework OK?

Yep, I tried smoketest only though.